packages/react-openapi/src/OpenAPISchema.tsx

```diff
@@ -1,3 +1,7 @@
+'use client';
+// This component does not use any client feature but we don't want to
+// render it server-side because it has recursion.
```

Comment:
Does it still mean that the first rendering on the client side will do all the recursion? or do we do them only when clicking/expanding?

It might impact client side perfs

Reply:
Rendering is only when we expand now.